cloudinit/handlers/shell_script_by_frequency.py

```diff
@@ -0,0 +1,88 @@
+# part-handler
+# vi: syntax=python ts=4
```

Comment:
Done .. but note those were lines were there because of https://cloudinit.readthedocs.io/en/latest/topics/format.html, so someone might wish to change the docs as well.

Reply:
Thanks for the feedback. It is silly for us to have that `#vi` comment in there (I'll put up a PR to get that removed). The `# part-handler` is valid for user-defined part handlers, but isn't necessary for this file because it is no longer a user-defined part handler, but becoming a built-in one.